src/main/java/cat/udl/eps/softarch/geolearning/domain/Game.java

```diff
@@ -22,6 +23,10 @@
 	@Length(min = 10, max = 250)
 	private String instructions;
 
+	@ManyToOne
+	@JsonIdentityReference(alwaysAsId = true)
+	private ContentCreator creator;
+
```

Comment:
PR opened for client side where this is used: [Image write](https://github.com/UdL-EPS-SoftArch/GeoLearning/pull/11#issue-422146328)